urdf/urdfdom_compatibility.h.in

```diff
@@ -69,16 +69,25 @@ URDF_TYPEDEF_CLASS_POINTER(Sphere);
 URDF_TYPEDEF_CLASS_POINTER(Visual);
 
 URDF_TYPEDEF_CLASS_POINTER(ModelInterface);
+
+typedef boost::shared_ptr<void> VoidSharedPtr;
```

Comment:
but we need to differentiate between `boost::shared_ptr` and `std::shared_ptr` depending on the urdfdom version.